README.md

```diff
@@ -10,29 +10,25 @@ Background job development in a breeze.
 # Installation
 Easy to install it via NuGet:
 
```

Comment:
This is confusing;
We have a first-level header 'installation', with directly underneath 3rd level headers.  

Then, we have a sentence which stops with a colon.   I would change that to 'The background jobs packages are easy to install via NuGet.'

Reply:
I suppose we will have a similar 'introduction' text on the documentation website ?

Reply:
On the docs site, we use the CloudEvents package as example and have the sentence: 'The Arcus BackgroundJobs can be installed via NuGet:'

Reply:
Ok, I'll change the headers and the sentence with your proposal! 👍